bin/rose-make-docs

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env bash
+#-----------------------------------------------------------------------------
+# (C) British Crown Copyright 2012-7 Met Office.
+#
+# This file is part of Rose, a framework for meteorological suites.
+#
+# Rose is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Rose is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Rose. If not, see <http://www.gnu.org/licenses/>.
+#-----------------------------------------------------------------------------
+# NAME
+#     rose make-docs
+#
+# SYNOPSIS
+#     rose make-docs [OPTIONS] [BUILD...]
+#
+# DESCRIPTION
+#     Build the rose documentation in the requested `BUILD` format(s).
+#
+# OPTIONS
+#     --venv
+#         Build virtualenv for temporarilly installing python dependencies
+#         if necessary.
+#     --dev
+#         Development mode, don't remove virtualenv after build.
+#
+# ENVIRONMENT VARIABLES
+#     SPHINX_DEV_MODE
+#         Set to `true` in order to make the `rose test-battery` command use
+#         development mode.
+#
+# BUILD
+#     The format(s) to build the documentation in - default html.
+#     Avaliable formats are listed in the sphinx documentation
+#     (http://www.sphinx-doc.org/en/stable/builders.html).
+#     The most commonly used formats are:
+#
+#     `html`
+#         For building standalone HTML files.
+#     `singlehtml`
+#         For building a single page HTML document.
+#     `latexpdf`
+#         For building PDF documentation.
+#
+# DEVELOPMENT BUILDS
+#     For development purposes use the following BUILDs:
+#
+#     `doctest`
+#         Runs any doctest examples present in documented python modules.
+#     `strict`
+#         Runs sphinx-build in a "strict" mode where cashing is turned off and
+#         warnings are turned into errors.
+#     `linkcheck`
+#         Checks external links.
+#
+#-----------------------------------------------------------------------------
+
```

Comment:
The only line I think would break is:

`virtualenv --python=python2.7 "${VENV_PATH}"`

You can do:

`virtualenv --python=python2.7 "${VENV_PATH:-}"`